web/src/utils/stringUtils.ts

```diff
@@ -1,8 +1,9 @@
 import normalizeStrings from 'normalize-strings'
 
-export const textTruncator = (text: string, numOfCharsAllowed: number): string => {
+export const textTruncator = (text: string, numOfCharsAllowed: number, replaceLineBreaks = true): string => {
```

Comment:
How do we handle this on native? Maybe we could move this to the api-client

Reply:
<ListItemView language={language}>
              <Content numberOfLines={5} language={language}>
                {content}
              </Content>
 </ListItemView>

```
export const Content = styled.Text<{ language: string }>`
  font-family: ${props => props.theme.fonts.native.decorativeFontRegular};
  font-size: 14px;
  letter-spacing: 0.5px;
  text-align: ${props => contentAlignment(props.language)};
  color: ${props => props.theme.colors.textColor};
`
```

Reply:
TextTrunctator is only used in web.

Reply:
in native we yould pass the number of lines which we sadly cannot pass in web